.github/workflows/repo-audit.yml

```
@@ -0,0 +1,202 @@
+            pyproject.toml
+            uv.lock
+
+      - name: Install hatch
+        run: uv tool install hatch==1.16.5
+
+      - name: Install pinned JS tooling (jscpd)
+        uses: pnpm/action-setup@008330803749db0355799c700092d9a85fd074e9
+        with:
+          version: 10.33.2
+          run_install:
+            args: [--frozen-lockfile]
```

actionlint (1.7.12)

[error] expected scalar node for string value but found mapping node with "!!map" tag

(syntax-check)